pkgs/applications/finance/odoo/default.nix

```diff
@@ -2,19 +2,74 @@
 , lib
 , fetchurl
 , python3
-, python3Packages
 , nodePackages
 , wkhtmltopdf
-, callPackage
 }:
 
-with python3Packages;
-
 let
-  werkzeug = python3Packages.callPackage ../../../development/python-modules/werkzeug/1.nix {};
-in
-
-buildPythonApplication rec {
+  python = python3.override {
+    packageOverrides = self: super: {
```

Comment:
how is this kept up to date? looks like a heavy extra maintaince burden

Reply:
The current solution doesn't work.

Reply:
My advice: ask upstream to move to a more recent Werkzeug.

Reply:
Ah, k. Then I'll merge this.